rendering/markup_render.go

```diff
@@ -3,35 +3,34 @@ package rendering
 import (
 	"regexp"
 
-	"github.com/almighty/almighty-core/workitem"
 	"github.com/microcosm-cc/bluemonday"
 	"github.com/russross/blackfriday"
 )
 
 // IsMarkupSupported indicates if the given markup is supported
 func IsMarkupSupported(markup string) bool {
 	if markup == "" || // assume same as default
-		markup == workitem.SystemMarkupDefault ||
-		markup == workitem.SystemMarkupMarkdown {
+		markup == SystemMarkupDefault ||
+		markup == SystemMarkupMarkdown {
 		return true
 	}
 	return false
 }
 
 // RenderMarkupToHTML converts the given `content` in HTML using the markup tool corresponding to the given `markup` argument
 // or return nil if no tool for the given `markup` is available, or returns an `error` if the command was not found or failed.
-func RenderMarkupToHTML(content, markup string) (*string, error) {
+func RenderMarkupToHTML(content, markup string) string {
```

Comment:
yes, that was something I did in the first commit of this PR but then I realized that there was never any error returned, so I decided to remove it in the second commit ;-)